fluidBuild.config.cjs

```diff
@@ -274,6 +274,7 @@ module.exports = {
 			"npm-package-exports-field": [
 				// We deliberately improperly import from deep in the package tree while we migrate everything into other
 				// packages. This is temporary and can be fixed once the build-tools/build-cli pigration is complete.
+				"^azure/",
```

Comment:
The 'npm-package-exports-field' logic is broken.  Nearly every package is was already exempt.  Just missing the '^azure/' path.

Things start to go wrong here in 'npmPackages.js':
```
			const isCJSOnly = json.module === undefined || json.type === "commonjs";
			const isESMOnly = json.type === "module";
```
...which gets into a state where both isCJSOnly and isESMOnly are both true simultaneously for a dual-format package.